Makefile

```diff
@@ -52,27 +52,28 @@ $(RUNSVINIT): vendor/runsvinit/*.go
 
 $(SCOPE_EXE): $(shell find ./ -path ./vendor -prune -o -type f -name *.go) prog/static.go $(CODECGEN_TARGETS)
 
-report/report.codecgen.go: $(call GET_CODECGEN_DEPS,report/)
+report/latest_map_impls_generated.go: report/latest_map_impls.go tools/generate_latest_map
+report/report.codecgen.go: $(call GET_CODECGEN_DEPS,report/) report/latest_map_impls_generated.go
```

Comment:
I had some errors during building `report.codecgen.go` about unknown type `StringLatestMap`, so I added the dep to make sure that this file with this type is generated before.